applications/asset_tracker/src/motion/motion.h

```diff
@@ -48,7 +48,8 @@ typedef struct {
 typedef struct {
 	motion_orientation_state_t orientation;
 	motion_acceleration_data_t acceleration;
-	/* TODO add timestamp */
+	/** Data sampling uptime. */
+	int64_t ts;
 } motion_data_t;
```

Comment:
This typedef must have slipped through the original review round on this module..

Reply:
Yes, I noticed that. Should be addressed in a future PR.